src/com/dogonfire/exams/Exams.java

```diff
@@ -20,9 +20,11 @@
 
 	public boolean				debug				= false;
 	public boolean				examPricesEnabled	= true;
+	public boolean				grantAdvancement	= false;
 
 	public String				serverName			= "Your Server";
 	public String				languageFilename	= "english.yml";
+	public String				examAdvacementName	= "doggycraft:doggycraft/student";
```

Comment:
This is a hardcoded advancement, for completing your first exam.
Would be better to move to the config file for the exams, to have custom per-exam advancements.

Or specify more precisely, that this advancement is a one-time advancement, by calling it something like "firstExamAdvancement".

Reply:
agree on naming but the "hardcoded" bit is just a default value. it is change later when the config is loaded.